src/Commands/ReapplyController.php

```
@@ -0,0 +1,143 @@
+
+            $anyUpdated = true;
+
+            $this->stdout(sprintf('[scaffold] Reapplied "%s".', $destination) . PHP_EOL);
+        }
+
+        if ($anyUpdated) {
+            $lock->write(['providers' => $data['providers'], 'files' => $updatedFiles]);
+        }
+
+        return ExitCode::OK;
+    }
```

PHPMD (2.15.0)

[warning] The method actionIndex() has a Cyclomatic Complexity of 16. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)